.github/workflows/release.yml

```diff
@@ -125,6 +125,11 @@ jobs:
             svm_target_platform: linux-aarch64
             platform: linux
             arch: arm64
+          - runner: Linux-22.04
```

Comment:
I think we should remove this because we already have the arm64 build

Reply:
ah yes this sneaked in again - will remove.